proxies/live/apiproxy/policies/AssignMessage.Remove.x-ers-business-function-header.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Are all these policy changes around headers tested in the Test_Headers test?

Reply:
I've added a test to cover this